src/docs/markdown/caddyfile/matchers.md

```diff
@@ -329,7 +329,7 @@ Match requests WITHOUT BOTH:
 - an `/api/` path prefix, AND
 - the `POST` request method
 
-i.e. can have zero or one of these to match
+i.e. can have neither or either of these to match:
```

Comment:
Would it be overcritical to request we change this to "must"?

I do very much like calling back to familiar wording and would like to keep the two as similar as possible so the difference is _exceptionally_ clear in comparison; "none" vs. "neither or either", rather than "must have none" vs "can have neither or either".

Reply:
I feel like `can` and `or` go hand-in-hand. I feel like `can` is the better word if there's multiple options, `must` being better if there's only one option

Reply:
Conversationally, `must` is exhaustive while `can` is permissive and open ended but not _strictly_ prohibitive beyond the stated options. It's ambiguous. If we're writing technically, "can" functions for both cases and we can ignore the conversational awkwardness because it's a technical statement. If we're writing conversationally, I think "must" is important in both cases.

Reply:
Fair enough.